### ISSUE TITLE
Workflow History (on overview) is removed on every deploy

### ISSUE THREAD

**Issue by kaylumah** [Azure/logicapps#405]
Hi,

We are using logic apps (standard) to automate some of our processes in sequence.
I noticed that on each deploy it removes all previous runs from the run history and the trigger history.

I was wondering if there is a way to keep these after new deploys and if I overlooked a setting.

Thanks
Max

**Comment by kaylumah** [comment]
Any update on this?

**Comment by puspraj-repos** [comment]
@kaylumah , I was trying to reproduce the issue, using default settings, when I'm using VS code to deploy the logic app, I don't see any history missing. Unless you delete the existing workflow, history should be there. can you share more detailed steps on how you are deploying the workflows?

**Comment by gastonmuijtjens** [comment]
Hi @puspraj-repos, thanks for your reply. I'm working on the same project as @kaylumah so I'm answering for him. We have found the issue why the workflow history was removed on every deploy. We have an automated release pipeline in Azure DevOps that automatically updates/creates the Logic App using ARM templates and uses the Azure Function Deploy task afterwards. 

We added the following configuration values to our ARM template in the `siteConfig/appSettings` section to resolve our issue:
```json
{
   "appSettings: [
       {
            "name": "WEBSITE_RUN_FROM_PACKAGE",
            "value": "1"
       }
   ]
}
```

Apparently omitting this setting from the ARM template caused the workflow history to be reset on each new incremental ARM deploy. This issue can be closed.
